src/services/kms/index.ts

```diff
@@ -123,3 +124,5 @@ export const producedDeployOutputTypes = [
 ];
 
 export const consumedDeployOutputTypes = [];
+
+export const supportsTagging = true;
```

Comment:
The Handel docs also don't show that Tags are supported for KMS, but we apparently do add tags to the stack. Can we update those docs as part of this PR?